app/Views/errors/html/error_404.php

```diff
@@ -74,7 +74,7 @@
 
 		<p>
 			<?php if (! empty($message) && $message !== '(null)') : ?>
-				<?= esc($message) ?>
+				<?= esc(str_replace(PHP_EOL, '<br/>', $message)) ?>
```

Comment:
Why not [nl2br](https://www.php.net/manual/en/function.nl2br.php)?

Reply:
Silly me. I forgot that. Sorry for reinventing the wheel. 🤣

Reply:
Fixed